src/foam/nanos/boot/NSpec.js

```diff
@@ -149,8 +149,8 @@ foam.CLASS({
     {
       class: 'Code',
       name: 'client',
-      shortName: 'c',
-      value: '{}'
+      value: '{}',
```

Comment:
I would rather just fix PROJECTION support. Where in clientBuilder is this happening?